pkg/plugins/autodiscovery/maven/main.go

```diff
@@ -65,6 +67,10 @@ func New(spec interface{}, rootDir, scmID string) (Maven, error) {
 		return Maven{}, err
 	}
 
+	if s.Mirror != "" {
+		maven.MavenCentralRepository = s.Mirror
```

Comment:
This will have unexpected side effects.
All Updatecli pipelines executed after this moment will use the mirror.

I think it's probably better to override the url in https://github.com/updatecli/updatecli/blob/9f9f34505083a8352c61a49e63206f981f550774/pkg/plugins/autodiscovery/maven/utils.go#L77